src/main/kotlin/no/fdk/dataservicecatalog/controller/ImportController.kt

```diff
@@ -0,0 +1,56 @@
+package no.fdk.dataservicecatalog.controller
+
+import no.fdk.dataservicecatalog.domain.ImportResult
+import no.fdk.dataservicecatalog.exception.OpenApiParseException
+import no.fdk.dataservicecatalog.handler.ImportHandler
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ProblemDetail
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.*
+import java.net.URI
+
+@RestController
+@RequestMapping(value = ["/import/{catalogId}"])
```

Comment:
for å holde samme logikk på endepunktene som ellers
```suggestion
@RequestMapping(value = ["/internal/catalogs/{catalogId}/import"])
```